README.md

```diff
@@ -96,7 +96,13 @@ const Circle = stampit(Point, {
   },
   methods: { // that goes to the prototype
     distance(point) {
-      return Point(point).distance(this) - this.radius;
+      // Shouldn't this only return a value if the point is located outside of the circle?
```

Comment:
Thanks for the tips mate.
This function returns positive value if the point is outside the circle. But returns negative value if the point is inside.